### PR TITLE
Caption updated by Mansimar

### PR DESCRIPTION
**Pull request by mansimar-kaur**
Updated caption

### CODE REVIEW

File: _slides/13-mansimar-kaur.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
 ---
 
-Caption here
+Commented by Mansimar Kaur
```
Comment (mansimar-kaur) [inline _slides/13-mansimar-kaur.md:6]: Looks good.

File: _slides/13-mansimar-kaur.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ layout: slide
 title: "Welcome to our slide deck!"
```
Comment (mansimar-kaur) [inline _slides/13-mansimar-kaur.md:3]:
```suggestion
title: "Welcome to our slide deck!"
```

**Review by martins-vds — APPROVED** [review]

File: _slides/13-mansimar-kaur.md [hunk 3]
```diff
@@ -1,9 +1,9 @@
 ---
 layout: slide
-title: "Welcome to our slide deck!"
+title: "Welcome SOF to our slide deck!"
 ---
 
-Caption here
+Commented by Mansimar Kaur
```
Comment (AlanGunadio) [inline _slides/13-mansimar-kaur.md:6]: wow very nice

**Review by AlanGunadio — COMMENTED** [review]
nice!

**Pull request merged**